pilot/lib/pilot/build_manager.rb

```diff
@@ -69,6 +69,11 @@ def distribute(options, build = nil)
         UI.success "Successfully set the changelog and/or description for build"
       end
 
+      if options[:beta_app_feedback_email].to_s.length > 0
+        build.update_build_information!(feedback_email: options[:beta_app_feedback_email])
```

Comment:
Shouldn't this also be set on line 68 with the others, so only one request is sent?